runner/steps.ts

```
@@ -161,7 +161,9 @@ function runBrowsers(context: Context) {
       // TODO(nevir): Better rationalize run-end and hook.
       await context.emitHook('cleanup');
 
-      return error;
+      if (error) {
+        throw error;
```

Are we okay throwing strings here? Can we throw an Error object instead?